.ruby-version

```diff
@@ -0,0 +1 @@
+1.9.3-p448
```

Comment:
I think that with Travis running the tests in different Rubies this should not be required. You can just as well argue that this should be JRuby 1.7.x because that way developers won't forget about JRuby support, etc.